run.py

```diff
@@ -17,7 +18,7 @@ def parse_arguments():
     parser.add_argument('--log_level', '-l', help='Logging level', default=DEFAULT_LOG_LEVEL)
     parser.add_argument('--format', '-f', help='Behave format', default=DEFAULT_BEHAVE_FORMAT)
     parser.add_argument('--feature_directory', '-fd', help='Feature directory', default=DEFAULT_FEATURE_DIRECTORY)
-    parser.add_argument('--show_skipped', help='Show skipped tests', action='store_true')
```

Comment:
I think we want to keep this, the logs of the non-regression runs get quite busy with all the skipped steps otherwise.